Rest.Services.Test/ProductRepositoryTest.cs

```diff
@@ -0,0 +1,70 @@
+using Xunit;
+using Rest.Models;
+using Rest.Services.Test.Utils.Fixtures;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Rest.Services.Test;
+
+public class ProductRepositoryTest
+{
+    [Fact]
+    public void Add_OneProduct_ShouldCountOneProduct()
+    {
+        var productRepository = new ProductRepository();
+        var product = new ProductFixtures().BuildProduct();
+        productRepository.Add(product);
+
+        Assert.Equal(1, productRepository.Count());
+    }
+
+    [Fact]
+    public void Remove_OneProduct_ShouldCountOneProducts()
+    {
+        var productRepository = new ProductRepository();
+        var products = new ProductFixtures().BuildProducts(2);

```

Comment:
Why is an instance method? I don't think instanced method is needed here.

Reply:
Good evening Reynold! What do you recommend to do here? I'm kind of clueless.

Reply:
Done! Why is it a better practice to do it?

Reply:
If the method does not need an instance, then one way to do it as such is using static methods instead

Reply:
Thank you!